timers.js

```diff
@@ -1,4 +1,35 @@
+$(document).ready(function() {
+  var seconds = 0;
+  var timerId;
+  var timeOn = false;
 
+  $("#start").on("click", function() {
+    console.log("clicked start");
+    if (timeOn == false) {
+      timeOn = true;
+      timerId = setInterval(function() {
+        seconds += 1;
+        $("#timer").text(seconds);
+      }, 1000);
+    }
+  });
 
+  $("#pause").on("click", function() {
+      if (timeOn == true) {
+      timeOn = false;
+      console.log("clicked pause");
+      clearInterval(timerId);
+    }
+  });
 
-
+  $("#reset").on("click", function() {
+        timeOn = false;
+        console.log("clicked reset");
+        clearInterval(timerId);
+        $("#timer").text("Stop Watch")
+        seconds = 0;
+      }
+    );
+  });
+//do the above functions have names? can I call the functions somehow? If not, can I add names to them?
```

Comment:
Are you referring to the callbacks being passed into your event listeners? Since they're anonymous functions you can't call them outside of the event listeners. There's nothing stopping you, however, from defining named functions and passing those as callbacks.

```js
// define the function
function startTimer( ){
  // code goes here
}

// pass it into the event listener
$("#start").on("click", startTimer);
```